src/modules/facility-covenant/dto/get-facility-covenants-response.dto.ts

```diff
@@ -0,0 +1,58 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ENUMS } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers/date-only-string.type';
+import { IsEnum } from 'class-validator';
```

Comment:
I don't think this adds anything because we're not validating the response

Reply:
Agreed, I've removed this both from this file and also from `deal-investor-response.dto.ts` (a GET response) where it also appears.